master/static/migrations/20220921224032_checkpoint-query-perf.tx.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE VIEW bun_checkpoints_view AS
+SELECT c.uuid::text                          AS uuid,
+       c.task_id,
+       c.allocation_id,
+       c.report_time,
+       'STATE_'::text || c.state             AS state,
+       c.resources,
+       c.metadata,
+       jsonb_build_object('trial_id', c.trial_id, 'experiment_id', c.experiment_id, 'experiment_config',
+                          c.experiment_config, 'hparams', c.hparams, 'training_metrics',
+                          jsonb_build_object('avg_metrics', c.training_metrics -> 'avg_metrics'::text, 'batch_metrics',
+                                             c.training_metrics -> 'batch_metrics'::text), 'validation_metrics',
+                          json_build_object('avg_metrics', c.validation_metrics), 'searcher_metric',
+                          c.searcher_metric) AS training,
+        c.experiment_id,
+        c.trial_id,
+        c.state AS state_enum
```

Comment:
this is just `proto_checkpoints_view` but with these three columns added. 
- `experiment_id` and `trial_id` so that we can filter before joining.
- `state_enum` so that we can filter by enum order within the db